File: algo/src/main/java/com/hartwig/actin/algo/medication/MedicationStatusInterpreter.java
package com.hartwig.actin.algo.medication;

import com.hartwig.actin.clinical.datamodel.Medication;

import org.jetbrains.annotations.NotNull;

public interface MedicationStatusInterpreter {

    @NotNull
    MedicationStatusInterpretation interpret(@NotNull Medication medication);
}

File: report/src/main/kotlin/com/hartwig/actin/report/pdf/tables/clinical/MedicationStatusInterpreter.kt
package com.hartwig.actin.report.pdf.tables.clinical

import com.hartwig.actin.clinical.datamodel.Medication

// TODO (CB): move this class and the copy in algo and the test fils to common module once common is converted to kotlin
interface MedicationStatusInterpreter {
    fun interpret(medication: Medication): MedicationStatusInterpretation
}